python/test/unit/language/test_core.py

```diff
@@ -3080,7 +3094,7 @@ def kernel(ptr, n_elements, num1, num2, type: tl.constexpr):
         err_msg = str(e)
 
     if type == "noinline":
-        assert err_msg is not ""
+        assert err_msg != ""
```

Comment:
did you mean to change this? let's revert this

Reply:
The original code breaks in my Python3.9 env. It has errors `SyntaxWarning: "is not" with a literal. Did you mean "!="?`. I feel it would be better to keep this change?